input/includes/value-sets.md

```diff
@@ -0,0 +1,401 @@
+  </tr>
+  <tr>
+    <td><a href="https://healthterminologies.gov.au/fhir/ValueSet/adverse-reaction-agent-1">Adverse Reaction Agent</a></td>
+    <td>StructureDefinition-au-core-allergyintolerance</td>
+    <td>ADHA</td>
+  </tr>
+  <tr>
+    <td><a href="https://hl7.org/fhir/R4/valueset-allergyintolerance-clinical.html">AllergyIntoleranceClinicalStatusCodes</a></td>
+    <td>StructureDefinition-au-core-allergyintolerance</td>
+    <td>HL7</td>
+  </tr>
+  <tr>
+    <td><a href="http://hl7.org/fhir/R4/valueset-reaction-event-severity.html">AllergyIntoleranceSeverity</a></td>
+    <td>StructureDefinition-au-core-allergyintolerance</td>
+    <td>HL7</td>
+  </tr>
+  <tr>
+    <td><a href="http://hl7.org/fhir/R4/valueset-allergyintolerance-verification.html">AllergyIntoleranceVerificationStatusCodes</a></td>
+    <td>StructureDefinition-au-core-allergyintolerance</td>
+    <td>HL7</td>
+  </tr>
+  <tr>
+    <td><a href="https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-amt-mp-codes.html">AMT Medicinal Product and Substances</a></td>
+    <td>StructureDefinition-au-core-medication</td>
+    <td>HL7 AU</td>
+  </tr>
+  <tr>
+    <td><a href="https://healthterminologies.gov.au/fhir/ValueSet/australian-immunisation-register-vaccine-1">Australian Immunisation Register Vaccine</a></td>
+    <td>StructureDefinition-au-core-immunization</td>
+    <td>Services Australia</td>
+  </tr>
+  <tr>
+    <td><a href="https://healthterminologies.gov.au/fhir/ValueSet/australian-indigenous-status-1">Australian Indigenous Status</a></td>
+    <td>StructureDefinition-au-core-patient</td>
+    <td>ADHA</td>
+  </tr>
+  <tr>
+    <td><a href="https://healthterminologies.gov.au/fhir/ValueSet/australian-medication-1">Australian Medication</a></td>
+    <td>StructureDefinition-au-core-medication, StructureDefinition-au-core-medicationstatement, StructureDefinition-au-core-medicationrequest</td>
+    <td>ADHA</td>
+  </tr>
+  <tr>
+    <td><a href="https://healthterminologies.gov.au/fhir/ValueSet/amt-vaccine-1">Australian Medicines Terminology Vaccine</a></td>
+    <td>StructureDefinition-au-core-medication, StructureDefinition-au-core-medicationstatement, StructureDefinition-au-core-medicationrequest</td>
+    <td>ADHA</td>
+  </tr>
+  <tr>
+    <td><a href="https://www.healthterminologies.gov.au/integration/R4/fhir/ValueSet/australian-pronouns-1">Australian Pronouns</a></td>
+    <td>StructureDefinition-au-core-patient</td>
+    <td>ADHA</td>
+  </tr>
+  <tr>
+    <td><a href="https://healthterminologies.gov.au/fhir/ValueSet/australian-states-territories-2">Australian States and Territories</a></td>
+    <td>StructureDefinition-au-core-patient, StructureDefinition-au-core-location, StructureDefinition-au-core-organization</td>
+    <td>ADHA</td>
+  </tr>
+  <tr>
+    <td><a href="https://healthterminologies.gov.au/fhir/ValueSet/australian-veteran-status-1">Australian Veteran Status</a></td>
+    <td>StructureDefinition-au-core-patient</td>
+    <td>ADHA</td>
```

Comment:
Disagree - NCTS covnentions have put copyright as ADHA. This is HL7 AU.

Remove this complex view. Make it a simple list.

Reply:
Agreed but this file has been committed by mistake and is not used anywhere. It has now been removed from this PR.

terminology.md has been updated to list all terminology associated with MS elements in AU Core (this resource is not included as the element it's bound to is not MS).